### DISCUSSION TITLE
[QST]Whether Spark-Rapids supports reading orc+zstd/parquet+zstd.

### DISCUSSION THREAD

**Discussion by lw-cxn** [NVIDIA/spark-rapids#6571] [Q&A] [answered]
Whether Spark-Rapids supports reading orc+zstd/parquet+zstd

**Comment by tgravescs** [comment, marked as answer]
yes we support zstd for reading for orc and parquet since the 22.08 release.
zstd is supported in Apache spark 3.2.0+ versions
